api/grpcserver/globalstate_service.go

```diff
@@ -448,13 +475,29 @@ func (s GlobalStateService) GlobalStateStream(in *pb.GlobalStateStreamRequest, s
 		channelLayer   chan events.NewLayer
 	)
 	if filterAccount {
-		channelAccount = events.GetAccountChannel()
+		if s.accountChannel == nil {
+			channelAccount = events.SubscribeToAccounts(30)
+		} else {
+			channelAccount = s.accountChannel
```

Comment:
This channel may be used multiple places, by multiple streams. I know this is only for debugging, but it could cause unintended errors - same as we're seeing now, where multiple subscribers to the same channel will not see everything published to the channel. The simplest fix might be to create a different struct member for each stream, e.g., for this stream rename `s.accountChannel` to `s.globalStateStreamAccountChannel`.